onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.cc

```diff
@@ -384,6 +384,12 @@ common::Status NnapiExecutionProvider::Compile(const std::vector<FusedNodeAndGra
           void* output_buffer = nullptr;
           size_t output_buffer_byte_size;
           if (!is_dynamic_shape_output) {
+            // Since NNAPI use {1} tensor as scalar, if the model output should have empty shape
+            // We are going to replace the {1} shape of the output back to {}
+            bool is_scalar_output = model->IsScalarOutput(output_name);
+            if (is_scalar_output)
```

Comment:
Add_Broadcast_0x0 should hit this code right? 

I look forward to a day when we can just run code coverage to confirm :)

Reply:
Yes, it is, if NNAPI EP is not excluded, then it will hit here